amazon-dynamodb-quickstart/README.md

```diff
@@ -0,0 +1,127 @@
+
+`./mvnw clean install -Pnative`
+
+and run with:
+
+`./target/amazon-dynamodb-quickstart-1.0-SNAPSHOT-runner` 
+
+
+# Running native in container
+
+Build a native image in container by running:
+
+`./mvnw package -Pnative -Dnative-image.docker-build=true`
+
+Build a docker image:
+`docker build -f src/main/docker/Dockerfile.native -t quarkus/amazon-dynamodb-quickstart .`
+
+Create a network that connect your container with localstack
+`docker network create localstack`
+
+Stop your localstack container you started at the begining
+`docker stop local-dynamo`
+
+Start localstack and connect to the network
+`docker run --rm --network=localstack --name localstack -p 8008:4569 -e SERVICES=dynamodb -e START_WEB=0 -d localstack/localstack`
+
+Create Dynamo table
+```
+aws dynamodb create-table --table-name QuarkusFruits \
+                          --attribute-definitions AttributeName=fruitName,AttributeType=S \
+                          --key-schema AttributeName=fruitName,KeyType=HASH \
+                          --provisioned-throughput ReadCapacityUnits=1,WriteCapacityUnits=1 \
+                          --profile localstack --endpoint-url=http://localhost:8000
+```
+
+Run quickstart container connected to that network (note that we're using internal port of the localstack)
+`docker run -i --rm --network=localstack -p 8080:8080 quarkus/amazon-dynamodb-quickstart -Dquarkus.dynamodb.endpoint-override=http://localstack:4569`
+
+Go to [`http://localhost:8081/fruits.html`](http://localhost:8081/fruits.html) or [`http://localhost:8081/async-fruits.html`](http://localhost:8081/async-fruits.html)
+
+# Using AWS account
+
+Before you can use the AWS SDKs with DynamoDB, you must get an AWS access key ID and secret access key. 
+For more information, see [Setting Up DynamoDB (Web Service)](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/SettingUp.DynamoWebService.html).
+
+Create a DynamoDB table using AWS CLI and your default AWS profile:
+
+```
+aws dynamodb create-table --table-name QuarkusFruits \
+                          --attribute-definitions AttributeName=fruitName,AttributeType=S \
+                          --key-schema AttributeName=fruitName,KeyType=HASH \
+                          --provisioned-throughput ReadCapacityUnits=1,WriteCapacityUnits=1
+```
+
+## Run demo
+
+You can run the demo the same way as for a local instance, but you need to change the `application.properties`.
+
+- remove or comment out `quarkus.dynamodb.endpoint-override` - as you are going to communicate with the AWS service now
+- remove or comment out `quarkus.dynamodb.aws.region` - region is going to be retrieve via the default providers chain in the following order:
+    - `aws.region` system property
+    - `region` property from the profile file
+- set `quarkus.dynamodb.aws.credentials.type=default` - to use the `default` credentials provider chain that looks for credentials in this order:
```

Comment:
I've also noticed when I comment out (do not provide) the credentials.type it will assume `default`.

Arguably all properties can be commented out?

Reply:
You're absolutely right. Looks like it's left over from the very initial instructions. will fix that.